MIGRATION.md

```diff
@@ -1486,734 +1704,624 @@ import {
+```tsx static=true
+<ToggleButton isActive icon={<CalendarIcon />}>
+  Embed Entry
+</ToggleButton>
+```
 
-<Spinner variant="default" size="medium" />;
+#### How to migrate your ToggleButton components
 
-<Spinner variant="primary" customSize={12} />;
+You must manually migrate the version 3 `ToggleButton` component updating icon prop. For example:
 
-<Spinner size="large" />;
+```tsx static=true
+<ToggleButton isActive icon="Calendar">
+  Embed Entry
+</ToggleButton>
+<ToggleButton isActive icon="Plus">
+  Add Entry
+</ToggleButton>
+```
 
-<Spinner size="small" className="test-class-name" />;
+becomes:
+
+```tsx static=true
+<ToggleButton isActive icon={<CalendarIcon />}>
+  Embed Entry
+</ToggleButton>
+<ToggleButton isActive icon={<PlusIcon />}>
+  Add Entry
+</ToggleButton>
 ```
 
-## Skeleton
+### Tooltip
 
-In v4 SkeletonContainer, we changed prop `animate` to `isAnimated` to align with our code [style guide](https://github.com/contentful/forma-36/blob/forma-v4/docs/code-style-guide.md). The rest stayed the same.
+In version 4, we renamed two Tooltip props: `containerElement` is now `as` and `place` is now `placement`.
 
-### How to migrate your Skeleton components
+#### How to migrate your Tooltip components
 
-To migrate your `Skeleton` component to v4, run the following [codemod](https://github.com/contentful/forma-36/tree/forma-v4/packages/forma-36-codemod):
+To migrate your `Tooltip` component to v4, run the following [codemod](https://github.com/contentful/forma-36/tree/forma-v4/packages/forma-36-codemod):
 
 `npx @contentful/f36-codemod`
 
-If you want to do it manually, you just need to update imports and rename prop `animate` to `isAnimated`:
+If you want to do it manually, you must transform your existing code as follows:
 
 ```tsx
-import {
-  SkeletonContainer,
-  SkeletonDisplayText,
-  SkeletonText,
-  SkeletonBodyText,
-  SkeletonRow,
-  Table,
-  TableBody,
-  SkeletonImage,
-} from '@contentful/forma-36-react-components';
+import { Tooltip } from '@contentful/forma-36-react-components';
 
-<SkeletonContainer animate={false}>
-  <SkeletonDisplayText numberOfLines={1} />
-  <SkeletonBodyText numberOfLines={3} offsetTop={35} />
-</SkeletonContainer>;
+<Tooltip content="content of the Tooltip" containerElement="div" place="left">
+  <TextLink>Hover me</TextLink>.
+</Tooltip>;
+```
 
-<Table>
-  <TableBody>
-    <SkeletonRow />
-  </TableBody>
-</Table>;
+Into:
 
-<SkeletonContainer>
-  <SkeletonImage />
-  <SkeletonText />
-</SkeletonContainer>;
+```tsx
+import { Tooltip } from '@contentful/f36-components';
+
+<Tooltip content="content of the Tooltip" as="div" placement="left">
+  <TextLink>Hover me</TextLink>.
+</Tooltip>;
 ```
 
-The result should be like this:
+### Workbench
+
+Workbench in version 4 got moved into a separate package, but since we would like to deprecate this component, we don't include it in the main package. In order to use this component, you need to install it from a separate package:
```

Comment:
```suggestion
In version 4, Workbench got moved into a separate package. Because this component will eventually be deprecated, we did not include it in the main package. To use this component you must install it from a separate package:
```